app/code/Magento/ImportExport/Test/Mftf/ActionGroup/AdminAssertVisiblePagerActionGroup.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertVisiblePagerActionGroup">
+        <see selector=".admin__data-grid-header .admin__control-support-text" userInput="records found" stepKey="seeRecordsLabel"/>
```

Comment:
Please, take a look at the existing elements for admin grids https://github.com/magento/magento2/blob/acf8f9ff281a0fb097ed40b8e58e16abbfec48c9/app/code/Magento/Ui/Test/Mftf/Section/AdminGridControlsSection.xml#L9
They are pretty common and that's a chance we can reuse a selector from there. If not, please, create a separate selector instead of hardcoding it directly in the action group

Reply:
Looks like the only available here is the `Per Page` dropdown selector - so we're able to use it here.
Also not sure about the pager selector, if it makes sense to move it in the mentioned by you Section, or leave as currently is.